chromadb/config.py

```diff
@@ -138,6 +138,8 @@ def chroma_server_auth_credentials_file_non_empty_file_exists(
     ] = "chromadb.auth.providers.RequestsClientAuthProtocolAdapter"
     chroma_client_auth_credentials_file: Optional[str] = None
     chroma_client_auth_credentials: Optional[str] = None
+    chroma_client_auth_token_transport_header: Optional[str] = None
```

Comment:
I really don't like how bloated flat Settings is getting. We should rethink this not now but next time we add to it.

Reply:
I agree. Split into sub-settings per module?

Reply:
in the future